src/clj/game/cards/programs.clj

```diff
@@ -856,8 +856,7 @@
                :async true
                :effect (req (if (= target "Done")
                               (do (shuffle! state side :deck)
-                                  (system-msg state side "shuffles the stack")
-                                  (effect-completed state side eid))
```

Comment:
This line needs to stay here. All branches of an async ability need to call effect-completed.